src/App.css

```diff
@@ -3,3 +3,9 @@
   color: white;
   background-color: #243e5d;
 }
+
+.dg_news_date-author p span:first-child {
+  border-right: 2px solid grey;
```

Comment:
Styles from the react-newsroom repo don't use the variables we have in the components repo, correct?

Reply:
correct. These are independent as of now. I think the goal is to eventually merge them but for now they are per "application"